CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ After installing necessary software for building provider from sources, you shou
 
 Make sure you have `$GOPATH/bin` in your `$PATH`:
 ```
-echo "export PATH=\$PATH:$(go env GOPATH)/bin" >> ~/.bash_profile
```

Comment:
Please revert this as it's incorrect. The goal of this line to generate a line like this in the `.bash_profile`:

```
export PATH=$PATH:/Users/<user>/exp/go/bin
```

but without `\` it will expand content of the `PATH` environment variable that is incorrect.